### PR TITLE
Added pbm tool

### PR DESCRIPTION
**Pull request by Aaronontheweb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Command           | Package Id | Author | Description
 `NuKeeper` | [NuKeeper](https://www.nuget.org/packages/NuKeeper) | [@AnthonySteele](https://github.com/anthonysteele) | Find outdated NuGet packages and apply updates to them.  [GitHub](https://github.com/NuKeeperDotNet/NuKeeper)
 `nyancat` | [nyancat](https://www.nuget.org/packages/nyancat/) | [@nickvdyck](https://github.com/nickvdyck) | Nyancat 😻 in your terminal, rendered through ANSI escape sequences. A port of the original terminal application to make this cat run on dotnet core. 🐱‍🏍  [GitHub](https://github.com/nickvdyck/nyancat.cs)
 `otterkeys` | [OtterKeys](https://www.nuget.org/packages/OtterKeys) | [@natsuo](https://github.com/natsuo) | Quickly create Ed25519 key pairs for signing and verifying messages or other data. [GitHub](https://github.com/miqo-no/OtterKeys)
+`pbm` | [Petabridge.Cmd](https://www.nuget.org/packages/pbm/) | [@petabridge](https://cmd.petabridge.com/) | CLI for managing [Akka.NET](http://getakka.net/) applications and clusters   [GitHub](https://github.com/petabridge/petabridge.cmd-issues)
```
Comment (natemcmaster) [inline README.md]:
```suggestion
`pbm` | [pbm](https://www.nuget.org/packages/pbm/) | [@petabridge](https://cmd.petabridge.com/) | CLI for managing [Akka.NET](http://getakka.net/) applications and clusters   [GitHub](https://github.com/petabridge/petabridge.cmd-issues)
```

Pbm is the package ID, right?

Comment (Aaronontheweb) [reply README.md]: PBM is the package ID, but Petabridge.Cmd is the commonly used name to refer to the tool. Named the package ID after the actual name of the .exe the user invokes.

Comment (Aaronontheweb) [reply README.md]: Ok, thanks!

Comment (amis92) [reply README.md]: @Aaronontheweb please change it. The current table entry is incorrect, as the PackageId *is* `pbm`. You may want to include the `Petabridge.Cmd` string in Description column.

**Pull request merged**